### ISSUE TITLE
Attaching HDMI monitor not working

### ISSUE THREAD

**Issue by ju2wheels** [stockmind/gpd-pocket-ubuntu-respin#30]
When I attempt to attach my monitor over HDMI, turning on the second monitor automatically messes up the resolution and view of the GPD Pocket screen. No matter what settings I choose it messes up the GPD Pocket screen and has a bunch of artifacts as well (resolution is not the only problem).

I was doing this off of Live USB and not on a full install so Im not sure if that would have made a difference.

**Comment by stockmind** [comment]
What kernel are you using? 

    uname -a

Are you using latest ISO on Mega or a new respinned one?

**Comment by ju2wheels** [comment]
```
$ md5sum gpdpocket-20170725-ubuntu-17.04-desktop-amd64.iso 
9e30f0a26c8e0fe73082a2500e6c8331  gpdpocket-20170725-ubuntu-17.04-desktop-amd64.iso
```
Ill have to check for the kernel later if you still need it. This is from the Mega link in the README.

**Comment by stockmind** [comment]
Attach a pic of the graphic issue.
Also what BIOS are you using?
Try to change this: /etc/X11/xorg.conf.d/20-intel.conf

    "DRI"   "true"

instead of the actual value.

**Comment by ju2wheels** [comment]
I figured out the problem. The first time you bring up the Display config window, both virtual displays are overalapping, so if you turn on the external display it will end up causing artifacts on both screens because the virtual displays are overlapping. Once I aligned them so they werent overlapping it works as expected.

https://imgur.com/a/Nf2R9

One weird thing I did notice, is that if I had the external display plugged in at boot, the external display becomes the primary for the boot sequence (GPD Pocket screen shows nothing) and it boots the Live USB all the way up to systemd starting services up and then both screens go black and you dont get any output. So booting with an external monitor plugged in doesnt seem to work.

Can close this otherwise, artifacts are due to the weird default virtual display alignment in the Display config.

**Comment by stockmind** [comment]
Thanks for the feedback and i should update Troubleshooting about this.
After both screens go black, your power led is white? If you press on it it
will complete boot?

**Comment by ju2wheels** [comment]
No, actually the lights on the power button go out completely fairly early in the boot process (during kernel init and before systemd services start coming up) when you boot with external monitor plugged in. If I press the power button it doesnt do much at all (short or long press), I pressed it long enough to see the systemd shutdown process but never saw any graphical output before the shutdown sequence.

I tested the same behavior with Windows 10 and booting with an external monitor completely jacks up the resolution on that too (the GPD screen comes up rotated counterclockwise like GPD's Ubuntu beta and the external monitor has the resolution of the GPD screen but vertical like a little tablet on your monitor).

Also I am on the 06/28 unlocked BIOS.
